go-sdk/pkg/sdk/keptn.go

```diff
@@ -268,14 +268,14 @@ func (k *Keptn) createFinishedEventForTriggeredEvent(triggeredEvent cloudevents.
 		genericEvent["result"] = "pass"
 	}
 
-	finishedEventType := strings.TrimSuffix(triggeredEvent.Type(), ".triggered") + ".finished"
-	keptnContext, _ := triggeredEvent.Context.GetExtension(KeptnContextCEExtension)
+	finishedEventType, _ := ReplaceEventTypeKind(receivedEvent.Type(), "finished")
```

Comment:
I think we should react on possible errors returned by this function (also in the line below)